EXILED/Exiled.API/Interfaces/IWrapper.cs

```diff
@@ -7,14 +7,11 @@
 
 namespace Exiled.API.Interfaces
 {
-    using UnityEngine;
-
     /// <summary>
     /// Defines the contract for classes that wrap a base-game object.
     /// </summary>
     /// <typeparam name="T">The base-game class that is being wrapped.</typeparam>
     public interface IWrapper<T>
-        where T : MonoBehaviour
```

Comment:
why ?

Reply:
oh sorry i didn't read before i accept let me ask of hwy @Thundermaker300  made this like that